packages/legend-query-builder/src/components/fetch-structure/QueryBuilderResultModifierPanel.tsx

```diff
@@ -128,31 +129,14 @@ const ColumnSortEditor = observer(
         <div className="query-builder__projection__options__sort__sortby">
           {sortType.toLowerCase()}
         </div>
-        <DropdownMenu
-          content={
-            <MenuContent>
-              {Object.values(COLUMN_SORT_TYPE).map((op) => (
-                <MenuContentItem key={op} onClick={changeSortBy(op)}>
-                  {op.toLowerCase()}
-                </MenuContentItem>
-              ))}
-            </MenuContent>
-          }
-          menuProps={{
-            anchorOrigin: { vertical: 'bottom', horizontal: 'left' },
-            transformOrigin: { vertical: 'top', horizontal: 'left' },
-            elevation: 7,
-          }}
+        <button
+          className="query-builder__projection__options__sort__sortby--btn btn--dark"
+          tabIndex={-1}
+          onClick={changeSortBy}
+          title="Choose SortBy Operator..."
         >
-          <div
-            className="query-builder__projection__options__sort__sortby--btn"
-            tabIndex={-1}
-            title="Choose SortBy Operator..."
-          >
-            <SortIcon />
-          </div>
-        </DropdownMenu>
-
+          <SortIcon />
```

Comment:
Instead of using `SortIcon`, could we use an upward pointing arrow for ascending and a downward pointing arrow for descending?